pkgs/applications/audio/airwave/default.nix

```diff
@@ -14,11 +14,10 @@ let
   };
 
   vst-sdk = stdenv.mkDerivation rec {
-    name = "vstsdk368_08_11_2017_build_121";
-    src = requireFile {
-      name = "${name}.zip";
-      url = "http://www.steinberg.net/en/company/developers.html";
-      sha256 = "e0f235d8826d70f1ae0ae5929cd198acae1ecff74612fde5c60cbfb45c2f4a70";
```

Comment:
Looks like unrelated commit ;)

Reply:
Yeah, I will, I accidentally bundled it :)

Reply:
Done